src/drawing/__tests__/DrawingCreator-test.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { mount, ReactWrapper } from 'enzyme';
-import DrawingCreator, { defaultStroke, Props } from '../DrawingCreator';
+import DrawingCreator, { Props } from '../DrawingCreator';
 import DrawingPath from '../DrawingPath';
 import DrawingSVG from '../DrawingSVG';
 import PointerCapture, { Props as PointerCaptureProps } from '../../components/PointerCapture';
+import { defaultStroke } from '../../store';
```

Comment:
We should be wary about importing from modules the component doesn't actually depend on. Should we have a separate "mock stroke" for testing purposes?